scala/Scala.g4

```diff
@@ -592,6 +595,15 @@ Delim
    : '`' | '\'' | '"' | '.' | ';' | ','
    ;
 
+Semi
```

Comment:
Maybe name it as `Sep`? `Semi` stand for semicolon.

Reply:
Yep, logical, but in [spec](https://scala-lang.org/files/archive/spec/2.13/13-syntax-summary.html) used this definition for `Semi`
```
semi  ::=  ‘;’ |  nl {nl}
```

If you want, I can change.

Reply:
> Yep, logical, but in [spec](https://scala-lang.org/files/archive/spec/2.13/13-syntax-summary.html) used this definition for `Semi`
> 
> ```
> semi  ::=  ‘;’ |  nl {nl}
> ```
> 
> If you want, I can change.

Let's follow the spec.